packages/tupaia-web/src/features/Map/utils/useMapOverlayTableData.ts

```diff
@@ -49,6 +51,14 @@ const useMapOverlayEntities = (
         includeRootEntity,
         filter: {
           type: measureLevel,
+          ...Object.entries(entityAttributesFilter).reduce((result, [attribute, filterValue]) => {
+            return {
+              ...result,
+              [`attributes_${attribute}`]: Array.isArray(filterValue)
+                ? filterValue.join(',')
+                : filterValue,
+            };
+          }, {}),
```

Comment:
Do you know if useEntities supports passing in a nested object? We handle all this manipulating of the filter in the `EntityApi` already, so if we can just pass in something like:
```suggestion
          ...Object.entries(entityAttributesFilter).reduce((result, [attribute, filterValue]) => {
            return {
              ...result,
              [`attributes_${attribute}`]: filterValue,
            };
          }, {}),
```

Reply:
It sort of formats it a bit weird if we pass in an array without joining it, which makes it return incorrect results. I have added a comment to this to make it clear why we need to do this.